### PR TITLE
docs: read through and fix minor problems 03

### PR DESCRIPTION
**Pull request by ericyangpan**

### CODE REVIEW

File: docs/en/04-concept/index.md [hunk 1]
```diff
@@ -6,128 +6,127 @@ In order to explain the basic concepts and provide some sample code, the TDengin
...
 
 ## Label/Tag
 
-Label/Tag refers to the static properties of sensors, equipment or other types of data collection devices, which do not change with time, such as device model, color, fixed location of the device, etc. The data type can be any type. Although static, TDengine allows users to add, delete or update tag values at any time. Unlike the collected metric data, the amount of tag data stored does not change over time. In the meters example, `location` and `groupid` are the tags.
+Label/Tag refers to the static properties of sensors, equipment or other types of data collection devices, which do not change with time, such as device model, color, fixed location of the device, etc. The data type can be any type. Although static, TDengine allows users to add, delete or update tag values at any time. Unlike the collected metric data, the amount of tag data stored does not change over time. In the meters example, `Location` and `Group ID` are the tags.
```
Comment (sangshuduo) [inline docs/en/04-concept/index.md]: Demo data's two tags are generated by taosBenchmark, and their name are "location" and "grouped"

File: docs/zh/04-concept/index.md [hunk 2]
```diff
@@ -4,119 +4,118 @@ title: 数据模型和基本概念
...
+      <td>2</td>
+    </tr>
+  </tbody>
 </table>
-<a href="#model_table1">表 1：智能电表数据示例</a>
+<a name="#model_table1">表 1. 智能电表数据示例</a>
 </div>
 
-每一条记录都有设备 ID，时间戳，采集的物理量以及每个设备相关的静态标签。每个设备是受外界的触发，或按照设定的周期采集数据。采集的数据点是时序的，是一个数据流。
+每一条记录都有设备 ID、时间戳、采集的物理量（如上表中的 Current，Voltage，Phase）以及每个设备相关的静态标签（Location 和 Group ID）。每个设备是受外界的触发，或按照设定的周期采集数据。采集的数据点是时序的，是一个数据流。
```
Comment (sangshuduo) [inline docs/zh/04-concept/index.md]: same comment as early

File: docs/zh/04-concept/index.md [hunk 3]
```diff
@@ -4,119 +4,118 @@ title: 数据模型和基本概念
...
-## 采集量 (Metric)
+## 采集量（Metric）
 
 采集量是指传感器、设备或其他类型采集点采集的物理量，比如电流、电压、温度、压力、GPS 位置等，是随时间变化的，数据类型可以是整型、浮点型、布尔型，也可是字符串。随着时间的推移，存储的采集量的数据量越来越大。智能电表示例中的电流、电压、相位就是采集量。
 
-## 标签 (Label/Tag)
+## 标签（Label/Tag）
 
-标签是指传感器、设备或其他类型采集点的静态属性，不是随时间变化的，比如设备型号、颜色、设备的所在地等，数据类型可以是任何类型。虽然是静态的，但 TDengine 容许用户修改、删除或增加标签值。与采集量不一样的是，随时间的推移，存储的标签的数据量不会有什么变化。智能电表示例中的location与groupId就是标签。
+标签是指传感器、设备或其他类型采集点的静态属性，不是随时间变化的，比如设备型号、颜色、设备的所在地等，数据类型可以是任何类型。虽然是静态的，但 TDengine 容许用户修改、删除或增加标签值。与采集量不一样的是，随时间的推移，存储的标签的数据量不会有什么变化。智能电表示例中的 `Location` 与 `Group ID` 就是标签。
```
Comment (sangshuduo) [inline docs/zh/04-concept/index.md]: same comment as early

**Review by sangshuduo — COMMENTED** [review]
see my comments

File: docs/en/04-concept/index.md (hunk 1 above)
Comment (ericyangpan) [reply docs/en/04-concept/index.md]: got it

**Review by sangshuduo — APPROVED** [review]
LGTM

**Pull request merged**